### DISCUSSION TITLE
Environments and namespaces creation

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#96] [Q&A] [answered]
Hi all!

When I create a new environment from the "Project Settings" area, does the Console create the corresponding K8S namespace as well?

If so, is there a way to automatically have some network policies applied on every environment of my company?

Thanks!

**Comment by davidebianchi** [comment, marked as answer]
Hi @Lucascanna!

Yes, the namespace will be created by Console (if the correct permissions are given to the Console user service account). If not given the permissions, the project will be created but the namespace don't (this could be useful for some environment where the namespaces must be created following a specific acceptance path).

For the network policy, in our PaaS we use kyverno to automate this type of processes (we should add it to our docs). But this I think that it's not enabled in all the clusters. You should open a Service Request asking information about your customer and cluster (and you can request to setup the network policy as well).
